test/Test/HTTPure/RequestSpec.purs

```diff
@@ -27,7 +28,7 @@ fromHTTPRequestSpec =
       mock <- mockRequest'
       mock.headers ?= headers mockHeaders
     it "contains the correct body" do
-      mockBody <- mockRequest' >>= _.body >>> toString
+      mockBody <- mockRequest' >>= (_.body >>> toString)
```

Comment:
I think you can revert this whole file too now

Reply:
Reverted